### DISCUSSION TITLE
Is there an observed Interactive available from Light house

### DISCUSSION THREAD

**Discussion by VargheseVarampinakathu-at** [GoogleChrome/lighthouse#14512] [Q&A]
For metrics like FirstContentful paint, I see the json has the throttled-value and the observedFirstContentful paint i.e. the actual value.

Is there a way to get 'interactive'  without applying throttling, if so what should I search for in the json?
![image](https://user-images.githubusercontent.com/109981331/200702275-e93a81e5-276e-4a2c-8272-eaf5b9ec13ea.png)

**Comment by adamraine** [comment]
There isn't any `observedInteractive` because we don't get it for free when computing the actual `interactive` value. That being said we can probably compute the observed value without much extra work.
